src/content/content.ts

```diff
@@ -0,0 +1,66 @@
+const headerComponent = {
```

Comment:
why are these all stored together in their own space rather than having them be part of their relevant components?

Reply:
It makes it easier to manage content on the app and we don't have to mix text with JSX/HTML which might make it look all over the place